pages/deployment/[uuid].tsx

```diff
@@ -194,7 +194,7 @@ export default function Deployment() {
           <DialogContentText sx={{ backgroundColor: 'whitesmoke', p: 2 }}>
             <pre>
               <div>
-                # Login to Docker (your token can be found on the <Link href='/settings'>/settings</Link> page)
+                # Login to Docker (your token can be found on the <Link href='/settings'>settings</Link> page)
```

Comment:
This was intentional due to it being a URL reference.  Pointing to the `/settings` url.  Can be removed if confusing though.

Reply:
Makes sense, but I personally prefer this way as most people don't navigate to a page by manually typing out an address and instead go by the page name and a hyperlink - happy to concede this, though, just my thoughts!